client/coniksclient/internal/cmd/common.go

```diff
@@ -39,5 +40,5 @@ func loadConfigOrExit(cmd *cobra.Command) *client.Config {
 
 // append "\r\n" to msg and then write to terminal in raw mode.
 func writeLineInRawMode(term *terminal.Terminal, msg string) {
-	term.Write([]byte(msg + "\r\n"))
+	term.Write([]byte("<" + time.Now().Format("15:04:05.999999999") + "> " + msg + "\r\n"))
```

Comment:
This seems like a debugging feature in the test client, maybe we could add a `-d/--debug` flag to the `run` command?